providers/salesforce/bulk-write.go

```diff
@@ -0,0 +1,38 @@
+package salesforce
+
+import (
+	"context"
+	"fmt"
+)
+
+// LaunchBulkWrite starts Bulk Job to upsert records.
+// https://developer.salesforce.com/docs/atlas.en-us.api_asynch.meta/api_asynch/create_job.htm
+//
+// After creation inspect newly launched Bulk Job via:
+// * GetJobInfo
+// * GetJobResults
+// * GetSuccessfulJobResults.
+func (c *Connector) LaunchBulkWrite( //nolint:funlen,cyclop
```

Comment:
I understand the rationale behind calling it `Launch` since it's a job, I would still want to call this as `BulkWrite`, and the same applies for other methods. It needs to align with similar methods that we will want to introduce for other providers in the future.

Reply:
I thought that it could be the case. I will do that.
However, the job info, job results or job successful results are confusing. I mention those in the method documentation to explain how things are related, it is not apparent from the code.

Reply:
Changed: https://github.com/amp-labs/connectors/pull/1010/commits/f2c83f52a9b267202ce9586e71fe522d56452250